contracts/ens/cid.go

```diff
@@ -31,7 +31,7 @@ const (
 	nsSwarm = 0xe4
 
 	swarmTypecode = 0xfa //swarm manifest, see https://github.com/multiformats/multicodec/blob/master/table.csv
-	swarmHashtype = 0xd6 // BMT, see https://github.com/multiformats/multicodec/blob/master/table.csv
+	swarmHashtype = 0x1b // BMT, see https://github.com/multiformats/multicodec/blob/master/table.csv
```

Comment:
Does this need a comment change too? Still mentioned BMT

Reply:
done. thanks @karalabe